addons/binding/org.openhab.binding.gardena/src/main/java/org/openhab/binding/gardena/internal/GardenaSmartImpl.java

```diff
@@ -342,9 +337,10 @@ public void sendCommand(Device device, GardenaSmartCommandName commandName, Obje
         }
 
         if (command != null) {
+            stopRefreshThread();
```

Comment:
> OK, i removed the true parameter now

But this now applies for all occasions, e.g. also for the handler disposal - which should be as quick as possible and there it was fully ok to interrupt potentially long running HTTP requests...